app/services/offer_validations.rb

```diff
@@ -31,7 +32,7 @@ def conditions_length
   end
 
   def identical_to_existing_offer?
-    if application_choice.current_course_option == course_option && application_choice.offer.conditions_text.sort == conditions.sort
+    if application_choice&.current_course_option == course_option && application_choice&.offer&.conditions_text&.sort == conditions.sort
```

Comment:
this can probably be improves as we should avoid trigerring this unless the validation is called through the ChangeOffer service